src/test/scala/org/hammerlab/guacamole/readsets/io/TestInputFilters.scala

```diff
@@ -0,0 +1,30 @@
+package org.hammerlab.guacamole.readsets.io
+
+import org.hammerlab.guacamole.loci.parsing.ParsedLoci
+
+object TestInputFilters {
```

Comment:
👍  This was annoyingly kept around even though it was only used in the tests, but I wasn't up for refactoring all of those!